lib/swaggerValidator/ajvSchemaValidator.ts

```diff
@@ -77,7 +78,25 @@ export class AjvSchemaValidator implements SchemaValidator {
       const result: SchemaValidateIssue[] = [];
       const isValid = validateSchema.validate.call(ctx, data);
       if (!isValid) {
-        ajvErrorListToSchemaValidateIssueList(validateSchema.validate.errors!, ctx, result);
+        const errors = validateSchema.validate.errors!;
+        const newErrors = lodash.cloneDeep(errors);
```

Comment:
We don't want to impact all errors here and just move the re-validation logic to a signle method.

Reply:
The error alone does not fully determine whether it can be skipped, query parameter array's item should also be revalidated